specs/simple-serialize.md

```diff
@@ -219,14 +219,22 @@ The decoding requires knowledge of the type of the item to be decoded. When
 performing decoding on an entire serialized string, it also requires knowledge
 of the order in which the objects have been serialized.
 
-Note: Each return will provide ``deserialized_object, new_index`` keeping track
-of the new index.
+Note: Each return will provide:
+- `deserialized_object`
+- `new_index`
```

Comment:
1. Could you add the definition of `deserialized_object` and `new_index` in `Terminology` section?
2. And actually, maybe `Terminology` should be rename to `Variables and Functions`...
2. Also, rename `byte_order` to `byteorder`.